classes/Poll.php

```diff
@@ -247,4 +258,16 @@ public function getResponseCount() {
 
 		return $this->response_count;
 	}
+
+	/**
+	 * Get amount of people who have voted
+	 *
+	 * @return int
+	 */
+	public function getVoterCount() {
+		// Make sure the values have been populated
+		$this->fetchResponses();
+
+		return $this->voter_count;
+	}
```

Comment:
This new method allows you to get the number of voters instead of number of individual votes.